src/main/java/seedu/address/commons/Comparators.java

```diff
@@ -34,12 +34,49 @@ public int compare(Team o1, Team o2) {
      * @return a new Comparator for Teams.
      */
     public static Comparator<Team> rankByScoreReverse() {
+        return rankByScore().reversed();
+    }
+
+    /**
+     * Creates a new comparator used to sort teams by the number of participants
+     * they have in ascending order, for use to break ties.
+     *
+     * @return a new Comparator for Teams.
+     */
+    public static Comparator<Team> rankByParticipants() {
         return new Comparator<Team>() {
             @Override
             public int compare(Team o1, Team o2) {
-                int scoreTeam1 = o1.getScore().getScore();
-                int scoreTeam2 = o2.getScore().getScore();
-                return scoreTeam1 - scoreTeam2;
+                int participantsTeam1 = o1.getParticipants().size();
+                int participantsTeam2 = o2.getParticipants().size();
+                return participantsTeam2 - participantsTeam1;
+            }
+        };
+    }
+
+    /**
+     * Creates a new comparator used to sort teams by their score
+     * in ascending order, mainly for the loser board.
+     *
+     * @return a new Comparator for Teams.
+     */
+    public static Comparator<Team> rankByParticipantsReverse() {
+        return rankByParticipants().reversed();
+    }
+
+    /**
+     * Creates a new comparator used to sort teams by their id
+     * in ascending order, for use to break ties.
+     *
+     * @return a new Comparator for Teams.
+     */
+    public static Comparator<Team> rankById() {
+        return new Comparator<Team>() {
+            @Override
+            public int compare(Team o1, Team o2) {
+                int idTeam1 = o1.getId().getNumber();
+                int idTeam2 = o2.getId().getNumber();
+                return idTeam2 - idTeam1;
```

Comment:
Feels like there can be one central comparator but multiple comparing functions

Reply:
Oh what do you mean? Like one comparator compares all fields?

Reply:
Seems like all of them are comparing numbers, could possibly abstract it out to use different comparator functions with the same comparator interface. but since only a few, is okay. Unless you plan on writing more comparators not much actionables on this

Reply:
ahhhhh I see. I'll try it out thanks.

For now I wasn't planning on adding any more comparators because there arent really many fields to compare by. But your idea is nice, it will make it easier to expand.